exercises/F-logical-operators/exercise2.js

```diff
@@ -5,7 +5,33 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(num) {
```

Comment:
There is a way for you to write this function with just 1 line instead of using the `if` statement. Can you think of how to do this?